Material.Demo/Pages/ColorsDemo.axaml.cs

```diff
@@ -0,0 +1,17 @@
+using Avalonia.Controls;
+using Avalonia.Markup.Xaml;
+
+namespace Material.Demo.Pages;
+
+public partial class ColorsDemo : UserControl
+{
+    public ColorsDemo()
+    {
+        InitializeComponent();
+    }
+
+    private void InitializeComponent()
```

Comment:
Initialize component are generated automatically

Reply:
Ok, that was auto-generated. Will remove this